src/pages/studyView/table/MultiSelectionTable.tsx

```diff
@@ -324,7 +329,11 @@ export class MultiSelectionTable extends React.Component<
                         data.numberOfAlteredCases,
                         data.matchingGenePanelIds || [],
                         this.toggleModal,
-                        { marginLeft: cellMargin }
+                        {
+                            flexDirection: 'row-reverse',
+                            display: 'flex',
+                            marginRight: cellMargin,
+                        }
```

Comment:
Now I see this code segment is being copied 3 times already. Might be better to do a little refactoring and extract this code segment to a single variable/constant to prevent code duplication.

Reply:
really we should do this with a css class.   @aditygrg2 i think this would be quite easy to do.   i searched project for "row-reverse" and it only comes up in four places.  so that suggests that with a small amount of refactoring, we can accomplish this using a css class, which would be better for code quality and also optimization since this gets used on a lot of table cells.